### ISSUE TITLE
make initialization of previous/historic values more consistent

### ISSUE THREAD

**Issue by robertoostenveld** [eegsynth/eegsynth#332]
Some modules start up with errors or inappropriate values, e.g. for the y-axis scaling in the plotting, which is due to the initialization of the historic scale being suboptimal. 

It is quite easy to initialize the historic values with values that make it work immediately.

**Comment by robertoostenveld** [comment]
The suboptimal behaviour in the plotting is mostly visible when setting learning_rate to a very small value, e.g. 0.01. In that case the initial adjustment takes very long to converge to the desired vertical scale. 

I changed the code so that the initialization for the scaling is not set at 0, but at the first value that makes sense. That means that the initial adjustment is not needed any more.
